src/utils/Sort.js

```diff
@@ -727,3 +727,16 @@ export function countGroup(group) {
   }
   return group.length;
 }
+
+export function downloadSort(cards) {
+  var exportCards = [];
+  cards = sortDeep(cards, 'Color', 'CMC');
+  cards.forEach((firstGroup) => {
```

Comment:
can you use for(const .. of ..) for this instead of .forEach()? This is a style change we've recently been moving towards